vision_datasets/common/data_manifest.py

```diff
@@ -95,15 +99,35 @@ def __init__(self, id, img_path, width, height, labels):
             labels (list or dict):
                 classification: [c_id] for multiclass, [c_id1, c_id2, ...] for multilabel;
                 detection: [[c_id, left, top, right, bottom], ...];
-                image caption: [caption1, caption2, ...];
-                image_text_matching: [(text1, match (0 or 1), text2, match (0 or 1), ...)]
+                image_caption: [caption1, caption2, ...];
+                image_text_matching: [(text1, match (0 or 1), text2, match (0 or 1), ...)];
                 multitask: dict[task, labels]
+                image_matting: [mating1, matting2, ...]
+            label_file_paths (list): list of paths of the image label files. "label_file_paths" only works for image matting task.
         """
         self.id = id
         self.img_path = img_path
         self.width = width
         self.height = height
-        self.labels = labels
+        self._labels = labels
+        self.label_file_paths = label_file_paths
+        self.file_reader = FileReader()
+
+    @property
+    def labels(self):
+        if self._labels:
+            return self._labels
+        elif self.label_file_paths:
+            self._labels = []
+            for label_file_path in self.label_file_paths:
+                self._labels.append(np.asarray(Image.open(self.file_reader.open(label_file_path))))
```

Comment:
We need to call close() for an object returned from self.file_reader.open().
with self.file_reader.open(label_file_path) as f:
    label = np.asarray(Image.open(f))